### PR TITLE
My pull request for dev challenge 1

### PR DESCRIPTION
**Pull request by luannguyen1410**

### CODE REVIEW

**Comment by luannguyen1410** [comment]
Dev-challenge-02: I added the new line for index.html

File: docs/dev-profile/luan-dev-profile.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>My Profile</title>
```
Comment (barrycumbie) [inline docs/dev-profile/luan-dev-profile.html:4]: your name vs. "My"

File: docs/dev-profile/luan-dev-profile.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    </head>
+    <body>
+        <h3>About Me</h3>
+        <img src="me.jpg" width="320" height="300">
+        <p>
```
Comment (barrycumbie) [inline docs/dev-profile/luan-dev-profile.html:9]: need to roll these into a single page...no extra files for this PR

File: docs/dev-profile/luan-dev-profile.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>My Profile</title>
+    </head>
+    <body>
+        <h3>About Me</h3>
```
Comment (barrycumbie) [inline docs/dev-profile/luan-dev-profile.html:7]: are we 2 spaces per google style guide?

File: docs/dev-profile/luan-dev-profile.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        <p>
+            <b>Name</b>: Luan "Lou" Nguyen<br>
+            <b>Country</b>: Vietnam<br>
+            <b>Occupation</b>: UNA Senior Student<br>
+            <b>Major</b>: Information Technology | Concentration: Software Development<br>
+            <b>Expected Graduation</b>: May 10th, 2021 (Hope so!)<br>
+            <b>Email</b>: lnguyen6@una.edu<br>
+            <b>Contact</b>:    <br>
+            <a href="https://github.com/luannguyen1410" target="_blank">
+            <img src="github.png" width="50" height="50"></a>
```
Comment (barrycumbie) [inline docs/dev-profile/luan-dev-profile.html:18]: suggest putting your gh username in the text of the link. 
Img needs alt tag to be valid.

File: docs/dev-profile/luan-dev-profile.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+            In this semester, I'm expanding my skills with Angular and Typescript.
+        </p>
+        <h3>Topic discussed in class</h3>
+        <ul>
+            <li>Github: a Version Control Platform.</li>
+            <li>Git Command: fetch, pull, push, commit, clone</li>
+            <li>Git Bash</li>
+            <li>Terminal command review</li>
+        </ul>
+
```
Comment (barrycumbie) [inline docs/dev-profile/luan-dev-profile.html:46]: where Nu validator?

**Comment by barrycumbie** [comment]
- [ ] single web-ready page
- [ ] google style guide on code: go find google style guide! 2 space indent, for example
- [ ] valid & linked to Nu
- [x] dev name
- [ ] dev pic
- [ ] dev github profile
- [x] dev major, grad, year
- [x] dev blurb
- [x] dev extras (links, memes, resources, skills)
- [x] dev env: your dv box and config ide
- [x] submit as PR

**Comment by barrycumbie** [comment]
see my comments and resolve.

**Pull request closed without merging**